vanilla_installer/gtk/default-image.ui

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <template class="VanillaDefaultImage" parent="AdwBin">
+    <property name="halign">0</property>
+    <property name="valign">0</property>
+    <property name="hexpand">true</property>
+    <child>
+      <object class="GtkOverlay">
+        <property name="valign">3</property>
+        <child type="overlay">
+          <object class="GtkButton" id="btn_next">
+            <property name="margin-end">12</property>
+            <property name="margin-start">12</property>
+            <property name="icon-name">go-next-symbolic</property>
+            <property name="halign">2</property>
+            <property name="valign">3</property>
+            <property name="tooltip-text" translatable="yes">Next</property>
+            <property name="sensitive">false</property>
+            <style>
+              <class name="circular"/>
+              <class name="suggested-action"/>
+            </style>
+          </object>
+        </child>
+        <child>
+          <object class="AdwStatusPage" id="status_page">
+            <property name="icon-name">image-missing-symbolic</property>
+            <property name="title" translatable="yes">Custom Image</property>
+            <property name="description" translatable="yes">Enter the URL of the custom image here. Note that the Vanilla OS team is not responsible for any issues that arise from using a custom image. Only install images from people you trust, as using their images gives them full control over your host system.</property>
```

Comment:
I think the part about full control over the system is pretty important here to inform the user what a custom image can do. If you would leave something out I would rather make it:

> Enter the URL of the custom image below. Note that custom images have full access to your system. Only install images from authors you trust.

The part about VanillaOS not being responsible is kinda redundant here in my opinion since it's already implied with "authors you trust".

Reply:
Sounds good as well